### PR TITLE
feat(cli): add delete workflow run command

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by fsamin — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17877.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

File: cli/cdsctl/workflow_run.go [hunk 1]
```diff
@@ -213,3 +213,30 @@ func workflowRunManualRun(v cli.Values) error {
...
+	Name:    "run-delete",
+	Short:   "Delete a workflow run",
+	Aliases: []string{"remove", "rm"},
+	Ctx: []cli.Arg{
+		{Name: _ProjectKey},
+		{Name: _WorkflowName},
+	},
+	Args: []cli.Arg{
+		{
+			Name: "run-number",
```
[the remark this answers is not in the document]
Comment (sguiheux) [reply cli/cdsctl/workflow_run.go:227]: We have nothing to do that with args on the cli.
Like the others command ( workflow stop etc.), it's still easy to write a for loop in bash

**Review by yesnault — APPROVED** [review]

**Review by richardlt — APPROVED** [review]

File: cli/cdsctl/workflow_run.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (fsamin) [reply cli/cdsctl/workflow_run.go:227]: fake news !

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17894.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#17898.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔ 
* Package
  * Docker ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✘

**Review by fsamin — APPROVED** [review]

**Pull request merged**